lib/regaliator/configuration.rb

```diff
@@ -1,17 +1,20 @@
 module Regaliator
   class Configuration
     attr_accessor :api_key, :secret_key, :host, :open_timeout, :read_timeout,
-      :use_ssl, :proxy_host, :proxy_port, :proxy_user, :proxy_pass, :version
+                  :use_ssl, :proxy_host, :proxy_port, :proxy_user, :proxy_pass
 
     def initialize
+      # Authentication settings
       @api_key      = nil
       @secret_key   = nil
+
+      # API host settings
       @host         = nil
-      @version      = nil
       @open_timeout = 10
       @read_timeout = 60
       @use_ssl      = true
```

Comment:
It shouldn't be an option anymore our API supports only SSL now.

Reply:
Fixed